### ISSUE TITLE
Hippocampe and Elastic Search query

### ISSUE THREAD

**Issue by shehreyarahmedkohati** [CERT-BDF/Hippocampe#70]
### Work Environment

|---------------------------|--------------------
| OS version (server)       | Ubuntu 16.04 VM on Hypervisor ESXI 6.0
Binary Installation of TheHive, Cortex, Synapse. 

### Problem Description
I have a running TheHive(3.1.1), Cortex(2.1.1), ElasticSearch(5.6.x), Synapse(Office365) integration on a single Ubuntu VM. I want hippocampe to complete my TheHive Project Stack.  

According to the hippocampe installation guide, ES version 5.x is required. But TheHive needs a newer version and I have a perfectly running integration of the above components with ES 6.x. 

What's the best possible way to go about this without disturbing the existing system?

**Comment by saadkadhi** [comment]
This should rather go as a question on Gitter or on our user forum. I will answer it here though and close the 'issue' but feel free to continue the discussion if needed on one of the above-mentioned channels.

TL;DR you need to use a separate ES installation for Hippocampe, an outdated application that we would like to rewrite with the help of the community. If interested, contact @ninSmith.

Hippocampe is outdated and it needs a complete rewrite. That's one of the reasons we barely if not ever mention it during our talks, presentations and workshops.

We sadly do not have the resources to rewrite it by ourselves in the short term and we are seeking the help of the community to do so. Unlike TheHive and Cortex, Hippocampe is written in Python. The barrier to entry is thus far less significant for coding souls should they decide to help us out.

@ninSmith is in charge of the rewriting project which has not yet formally started. If you are interested in helping out and draw a roadmap with us, it's time to join the boat. Contact us at support@thehive-project.org. Thanks.

**Comment by shehreyarahmedkohati** [comment]
Thank you for the help
